lib/resources/r4/communication.yaml

```diff
@@ -189,5 +169,30 @@ fields:
         }
       ]
     }
-  action:
-    - create
+  action: create
+
+- name: Communication Email-Status Extension
+  terminology_name: extension[x].valueCodeableConcept
+  required: 'No'
+  type: CodeableConcept
+  description: Status extension for the Communication.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/email-status",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "code": "in-progress",
+            "system": "http://hl7.org/fhir/task-status"
+          }
+        ]
+      }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/email-status?_format=json
+  binding:
+    description: A status extension for the Communication.
```

Comment:
Is this the binding description that we have decided for email status?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/4aeb3415a4f9e7dd47d6f3c87b989b123d9fb1ff